app/components/contacts/CreateNewContact.js

```diff
@@ -119,8 +121,8 @@ class CreateNewContact extends Component<Props, State> {
           <InputWrapperLowerPart />
         </InputsWrapper>
         <ButtonsWrapper>
-          <Link onClick={onCancel} text="CANCEL" style={{ color: smColors.disabledGray, marginRight: 15 }} />
-          <Link onClick={this.createContact} text="CREATE" style={{ color: smColors.white }} />
+          <Button onClick={onCancel} text="CANCEL" style={{ color: smColors.disabledGray, marginRight: 15 }} />
+          <Button onClick={this.createContact} text="CREATE" style={{ color: smColors.white }} />
```

Comment:
According to design it should be link-like buttons so no change here

Reply:
i dont understand, does this mean we wrap a button inside a link like <Link><Button/></Link>? or should I just change the color of the link from smColors.white